nl/link_linux.go

```diff
@@ -70,6 +70,20 @@ const (
 )
 
 const (
+	IFLA_IPVLAN_UNSPEC = iota
+	IFLA_IPVLAN_MODE   = iota
+	IFLA_IPVLAN_MAX    = IFLA_IPVLAN_MODE
+)
+
+type IPVlanMode uint16
```

Comment:
This type should be in link.go since it is a higher level type used by the netlink library not the lower level nl library (see type Mode uint8 in xfrm.go for example).